<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsReportPage</class>
 <widget class="QWidget" name="ResultsReportPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>876</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,12,1">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,1">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="top_h_layout_" stretch="0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="results_button_">
            <property name="text">
             <string>Results</string>
            </property>
            <property name="icon">
             <iconset resource="../app_resources.qrc">
              <normaloff>:/icons/resources/icons/arrow_left.png</normaloff>:/icons/resources/icons/arrow_left.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="title_label_">
          <property name="styleSheet">
           <string notr="true">padding-left:8px;</string>
          </property>
          <property name="text">
           <string>MLPerf Client Results Report</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="top_rh_layout_">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Show Subtask Results</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="result_table_frame_">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1007</width>
           <height>648</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="2" rowspan="5">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item row="1" column="4" rowspan="4">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3" rowspan="3">
         <widget class="QPushButton" name="run_new_benchmark_button_">
          <property name="minimumSize">
           <size>
            <width>397</width>
            <height>63</height>
           </size>
          </property>
          <property name="text">
           <string>Run New Benchmark</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5" rowspan="4">
         <widget class="QPushButton" name="export_button_">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Export               </string>
          </property>
          <property name="icon">
           <iconset resource="../app_resources.qrc">
            <normaloff>:/icons/resources/icons/export_icon.png</normaloff>:/icons/resources/icons/export_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QLabel" name="TTFT_label_">
          <property name="text">
           <string>*TTFT = Time to first token</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="3">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="TPS_label1_">
            <property name="text">
             <string>*TPS = Average tokens per second from the</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TPS_label2_">
            <property name="text">
             <string>second token to the end of the sequence</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../app_resources.qrc"/>
 </resources>
 <connections/>
</ui>
